### ISSUE TITLE
What is expected result of the pretrained model?

### ISSUE THREAD

**Issue by 22nicolet** [romainloiseau/Helix4D#3]
Hi,

I think I've finished running the pretrained model, and it generates a Test Metric and DataLoader table. Is there any other expected output (e.g: accuracy metrics on classifiying test data)? 

Thank you!

**Comment by romainloiseau** [comment]
No their is no other expected output. You can however edit the implementation to output whatever you need.
